extensions-core/azure-extensions/src/test/java/org/apache/druid/data/input/azure/AzureInputSourceTest.java

```diff
@@ -214,6 +216,78 @@ public void test_toString_returnsExpectedString()
     Assert.assertEquals("AzureInputSource{uris=[], prefixes=[azure://container/blob], objects=[]}", actualToString);
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void test_Deny_All()
```

Comment:
Is this the same as the `testDenyAll()` test below?

Reply:
done.